docs/resources/azure_express_route_circuits.md

```diff
@@ -0,0 +1,115 @@
+---
+title: About the azure_express_route_circuits Resource
+platform: azure
+---
+
+# azure_express_route_circuits
+
+Use the `azure_express_route_circuits` InSpec audit resource to test properties of Azure ExpressRoute circuits for a resource group or the entire subscription.
+
+## Azure Rest API Version, Endpoint, And HTTP Client Parameters
+
+This resource interacts with API versions supported by the resource provider.
+The `api_version` can be defined as a resource parameter.
+If not provided, the latest version will be used.
+For more information, refer to [`azure_generic_resource`](azure_generic_resource.md).
+
+Unless defined, `azure_cloud` global endpoint, and default values for the HTTP client will be used.
+For more information, refer to the resource pack [README](../../README.md).
+
+## Availability
+
+### Installation
+
+This resource is available in the [InSpec Azure resource pack](https://github.com/inspec/inspec-azure).
+For an example `inspec.yml` file and how to set up your Azure credentials, refer to resource pack [README](../../README.md#Service-Principal).
+
+Also, refer to [Azure documentation](https://docs.microsoft.com/en-us/rest/api/expressroute/express-route-circuits/list) for the available properties.
+Any attribute in the response may be accessed with the key names separated by dots (`.`).
+
+## Syntax
+
+An `azure_express_route_circuits` resource block returns all Azure ExpressRoute circuits, either within a resource group or a subscription.
+
+```ruby
+describe azure_express_route_circuits(resource_group: 'RESOURCE_GROUP') do
+  #...
+end
+```
+
+or
+
+```ruby
+describe azure_express_route_circuits do
+  #...
+end
+```
```

Comment:
Can someone double-check that this Syntax section is correct.

Reply:
Hi Ian,

Syntactically its correct and test cases also works this way but I will remove this, as I again checked in Api documentation They expect this resource by resource group only not subscription level.
So I am removing the example from document.